graph/src/components/store.rs

```diff
@@ -128,7 +128,10 @@ impl EntityQuery {
             filter: None,
             order_by: None,
             order_direction: None,
-            range: None,
+            range: EntityRange {
+                first: 100,
+                skip: 0,
+            },
```

Comment:
Sorry i didn't catch this earlier, but this change is too intrusive: we also use `EntityQuery` for our subgraph metadata, and all such queries are now also restricted to 100 entries, even if we mean for them to be unrestricted. For example, we list all subgraphs [here](https://github.com/graphprotocol/graph-node/blob/master/core/src/subgraph/registrar.rs#L284) and if there are more than 100 subgraphs, they will be invisible.

We should go back to the way things were with the internals of `EntityQuery` and how the store handles them, and only restrict queries that were supplied by the user to the first 100 entities.

Reply:
Thanks for catching this, this would be awful! I'll fix it.